drivers/mspi/Kconfig.nrfe

```diff
@@ -25,4 +25,23 @@ config MSPI_NRFE_INIT_PRIORITY
 	  SDP MSPI initialization depends on IPC initialization
 	  which is done at the same init level and has init priority equal to 46.
 
+choice MSPI_NRFE_IPC_TRANSFER_MODE
```

Comment:
Nit: now, that I think about it, it could be just one Kconfig option, that would determine if no copy should be used or not, choice might be too much here. But it can be left as it is.